src/adaptationConfig.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Configures the adaptation logic.
+ */
+export interface AdaptationConfig {
+  /**
+   * The upper bitrate boundary in bits per second for approximate network bandwidth consumption of the played source.
```

Comment:
We had a brief discussion about with @matamegger that resulted in this API docs.
We found this describes the idea behind of the flag and the behavior the best on both platforms.

Context:
- on Android: this affects only video rendition bitrate selection
- on iOS: this affects all rendition bitrate selection

Reply:
Thanks for the context. I think we could even add this to the docs. What do you think about that?

Reply:
Additionally (to avoid confusion) we could also mention that (at least in Android) it's still possible to over this limit manually.